src/styles/pages/_search.scss

```diff
@@ -0,0 +1,34 @@
+.search-result {
+  border-block-end: solid 1px #000;
```

Comment:
One more missing colour token for `#000` and I swear after that it's good to go :D